modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-infra-adding-ingress_{context}"]
+= Optional: Adding the ingress DNS records
+
+If you removed the DNS Zone configuration when creating Kubernetes manifests and
+generating Ignition configs, you must manually create DNS records pointing at
```

Comment:
s/pointing/that point

I wish that we could share the same module: https://github.com/openshift/openshift-docs/pull/17190/files#diff-9fad88b2dfefb1e3021ffd383eaad12bR1